docs/components.md

```diff
@@ -186,7 +186,7 @@ However KubeVirt might define a Storage CRD along side with a flow description w
 ## Networking
 
 We will try to leverage as much of Kubernetes networking plugin mechanisms (e.g. CNI).
-However, `virt-handler` may provide a plugin mechanism to allow network setup on a host, if the KubeVirt requirements do not fit into the Kubernetes storage scenarios.
```

Comment:
Thanks for your fix @ChAnYaNG97 !
However, I feel that the entire sentence is out of date. With designed https://github.com/nmstate/kubernetes-nmstate exactly to avoid KubeVirt-specific host network setup.

@phoracek can we work to update this page with our current vision, which includes https://github.com/kubevirt/cluster-network-addons-operator ?

Reply:
Sure. I've opened https://github.com/kubevirt/kubevirt/issues/7066 to track this